index.html

```diff
@@ -411,7 +411,7 @@ <h1>Sponsors</h1>
       <div class="contact section text-center">
         <div class="container">
           <div class="contact-header">
-            <p>Contact Us</p>
+            <h1 style="font-size: 100%;">Contact Us</h1>
```

Comment:
nit: can we add a media query for 575px and add some padding-bottom to this header? right now it looks a little squished for smaller screens
![image](https://user-images.githubusercontent.com/41929993/91318480-45743180-e789-11ea-96ac-3a82a8886196.png)